arbitrum-docs/how-arbitrum-works/06-optimistic-rollup.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: Optimistic Rollup
+description: Learn the fundamentals of Arbitrum's optimistic rollup.
+author: pete-vielhaber
+sme: TucksonDev
+user_story: As a current or prospective Arbitrum user, I need to learn more about Arbitrum's Optimistic Rollup protocol.
+content_type: get-started
+---
+
+Arbitrum is an optimistic rollup. Let’s unpack that term.
+
+_Rollup_
+
+Arbitrum is a rollup, which means that the inputs to the chain -- the messages that are put into the inbox -- are all recorded on the Ethereum chain as calldata. Because of this, everyone has the information they would need to determine the current correct state of the chain -- they have the full history of the inbox, and the results are uniquely determined by the inbox history, so they can reconstruct the state of the chain based only on public information, if needed.
```

Comment:
```suggestion
Arbitrum is a Rollup, which means that the inputs to the chain -- the messages that are put into the inbox -- are all recorded on the Ethereum chain as calldata. Because of this, everyone has the information they would need to determine the current correct state of the chain -- they have the full history of the inbox, and the results are uniquely determined by the inbox history, so they can reconstruct the state of the chain based only on public information, if needed.
```